pyuvdata/uvbase.py

```diff
@@ -171,6 +171,15 @@ def __getattr__(self, __name):
                     elif __name == "telescope_location_lat_lon_alt_degrees":
                         ret_val = self.telescope._location.lat_lon_alt_degrees()
                 return ret_val
+        elif __name == "future_array_shapes":
+            warnings.warn(
+                f"The UVData.{__name} attribute is now deprecated, as all UVBase "
```

Comment:
Since this can be raised on UVCal and UVFlag objects as well as UVData objects, we should remove the `UVData` in this warning.

Reply:
Ah, fair point -- fixed in 2a03cbd5992b6a0dd2b5f76e8b8c856e8faa741f.